src/Compilers/VisualBasic/Portable/Compilation/SyntaxTreeSemanticModel.vb

```diff
@@ -864,7 +864,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' </summary>
         Private Function CheckSymbolLocationsAgainstSyntax(symbol As NamedTypeSymbol, nodeToCheck As VisualBasicSyntaxNode) As NamedTypeSymbol
             For Each location In symbol.Locations
-                If location.SourceTree Is Me.SyntaxTree AndAlso nodeToCheck.Span.Contains(location.SourceSpan.Start) Then
+                If location.SourceTree Is Me.SyntaxTree AndAlso nodeToCheck.Span.Contains(location.SourceSpan) Then
```

Comment:
Imagine a user creating a new partial class:
```
Partial Class $$
```
In the above, `location.SourceSpan.Start` will be 14 (the start of the missing identifier). As it is now, the call to `Contains(int)` treats the node span as a right-open interval [0, 14), so the check fails. Changing the code to use `Contains(Span)` - the span being [14,14) - returns `True` for this case.